package.json

```diff
@@ -269,6 +269,7 @@
     "elliptic": "^6.5.3",
     "nwmatcher": "^1.4.4",
     "http-proxy-agent": "^5.0.0",
-    "https-proxy-agent": "^5.0.0"
+    "https-proxy-agent": "^5.0.0",
+    "minimist": "1.2.6"
```

Comment:
Why do you need to add minimist here? That should just be a build folder thing, right?

Reply:
yeah, I can revert the last commit I made, since it didn't make any difference.